system/lib/gl/webgl2.c

```diff
@@ -153,6 +151,8 @@ GL_APICALL GLboolean GL_APIENTRY glIsVertexArrayOES(GLuint array) { return glIsV
 GL_APICALL void GL_APIENTRY glDrawBuffersEXT(GLsizei n, const GLenum *bufs) { glDrawBuffers(n, bufs); }
 GL_APICALL void GL_APIENTRY glDrawBuffersWEBGL(GLsizei n, const GLenum *bufs) { glDrawBuffers(n, bufs); }
 
+#endif // ~__EMSCRIPTEN_PTHREADS__) && __EMSCRIPTEN_OFFSCREEN_FRAMEBUFFER__
```

Comment:
Changing this #endif looks incorrect? The above extensions are available in WebGL all the time, even when not targeting pthreads or Offscreen Framebuffer?

Reply:
I agree, this is should not be part of this PR.

However, accoarding my analysis, these aliases are normally defined by JS:  

The only time we need these native wrappers AFAICT is when we are using proxying and we want to use the native-wrappers defined above.  For example `glDrawBuffersWEBGL` normally defined as an alias here:
https://github.com/emscripten-core/emscripten/blob/07af96af4a1fbfe293aa2fb6c64236f00afc201f/src/library_webgl.js#L3921.

There reason this native wrapper exists is so that we can intercept it and proxy it using the `glDrawBuffers` wrapper just above here: https://github.com/emscripten-core/emscripten/blob/07af96af4a1fbfe293aa2fb6c64236f00afc201f/system/lib/gl/webgl2.c#L31

Anyway, this is a separate change.

Reply:
These changes are part of a separate PR: #19046.   We can discuss them further there.